WORKSPACE

```diff
@@ -32,11 +32,11 @@ local_repository(
 )
 
 http_archive(
-    name = "com_github_bazelbuild_buildtools",
-    sha256 = "e3bb0dc8b0274ea1aca75f1f8c0c835adbe589708ea89bf698069d0790701ea3",
-    strip_prefix = "buildtools-5.1.0",
+    name = "buildifier_prebuilt",
```

Comment:
I'd like to approve this change, but I don't see many Google projects using this binary dependency.

@hzeller What do you think?

Reply:
We can apt-get a buildifier on the CI if the script/run_buildifier.sh is to be run there.

Reply:
I've updated the PR to install buildifier with `go install` (I couldn't find it in the apt repositories)